src/System.Collections/tests/Performance/Perf.Dictionary.cs

```diff
@@ -193,5 +193,356 @@ public void ContainsKey(int size)
                         dict.ContainsKey(key); dict.ContainsKey(key); dict.ContainsKey(key);
                     }
         }
+
+        [Benchmark]
+        [InlineData(1)]
+        [InlineData(10)]
+        [InlineData(1000)]
+        public static void Remove_ValueType(long size)
+        {
+            s_collection1 = new Dictionary<long?, long?>();
+            long?[] items;
```

Comment:
Not sure of the original intent here.  My guess is that we're testing some amount of nullable here, but I couldn't say for sure.